jenkins-config/managed-config-files/globalMavenSettings/global-settings/content

```diff
@@ -0,0 +1 @@
+../../../managed-config-templates/mavenSettings-content
```

Comment:
globalMavenSettings content is not the same as the standard mavenSettings content. You should take the regular boiler plate globalMavenSettings and remove all comments, but all of the sections that it had should remain (we've never tested to see what happens if they're removed!)

Reply:
done